Makefile

```diff
@@ -128,16 +129,22 @@ endif
 ifeq ($(RPM_REPOSITORY),oss)
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL7}
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
+
+	mock -r epel-8-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL8}
+	mock -r epel-8-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL8}
 else
 	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL7}
-	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
+	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --rebuild ${RPMDIR-EL8}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
+
+	mock -r epel-8-x86_64 --bootstrap-chroot --enablerepo="$(ISO_REPO)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL8}
```

Comment:
Does the `epel-8-x86-64` mock config contain the config for the new cloudsmith repos?

Reply:
I'm not very familiar with mock so I'm not sure what config you're talking about. I can tell you though that epel-8-x86_64 is used by the build job for inmanta-core as well.

Reply:
inmanta-and-epel-7-x86_64 is specific config that we deploy on the jenkins slaves so that the build can access the other already built RPMs

Reply:
Indeed. We should add a similar config for Centos 8. The `inmanta-and-epel-7-x86_64` is present in the infra model.

Reply:
Should I have a look at this now or do we ticketize this (I can't estimate whether this is a simple fix)?

Reply:
You will not be able to test the build without doing this

Reply:
I can assist if required

Reply:
I'll let you know, I'll have to read up on mock first.

Reply:
Am I correct this is required because `python3-inmanta` has `BuildRequires python3-inmanta-core`, so we need to make sure mock's chroot knows where to find that?

Reply:
As a result the inmanta-core release pipeline doesn't need this because it doesn't require any of our other RPMs.

Reply:
yes

Reply:
nope, but all the others do. So I would use the same mock config for all the RPMs.